### DISCUSSION TITLE
v5.1

### DISCUSSION THREAD

**Discussion by mgth** [mgth/LittleBigMouse#320] [Announcements]
- Fix daemon crash on start
- Add monitors list view mode

**Full Changelog**: https://github.com/mgth/LittleBigMouse/compare/v5.0-beta4.1...v5.0-beta5

## Donation
If this project help you, you can give me a cup of coffee :) 
[![Donate](https://img.shields.io/badge/Donate-PayPal-green.svg)](https://www.paypal.com/cgi-bin/webscr?cmd=_donations&business=YLGYPSHWTQ5UW&lc=FR&item_name=Mgth&currency_code=EUR&bn=PP%2dDonationsBF%3abtn_donateCC_LG%2egif%3aNonHosted)

<hr /><em>This discussion was created from the release <a href='https://github.com/mgth/LittleBigMouse/releases/tag/v5.1'>v5.1</a>.</em>

**Comment by ChristopherHaws** [comment]
Hmm, not sure what is different about this release from previous ones but Windows Defender is picking up this version as a virus.

Brave:
![image](https://github.com/mgth/LittleBigMouse/assets/5934505/e5b64e46-e859-4fd9-8e61-896a6e19f224)

Firefox:
![image](https://github.com/mgth/LittleBigMouse/assets/5934505/7accbe56-c472-4646-855e-6b3e420ee345)

Windows Defender
![image](https://github.com/mgth/LittleBigMouse/assets/5934505/2bd1d23a-eb62-41bb-96dd-5443953e09b2)

**Reply by abowman02** [reply]
Why do all of my antivirus software show me the .exe for this contains "W32.AIDetectMalware" in the form of Bkav Pro?

**Comment by mgth** [comment]
Sorry about that, I tried changing installer but nothing works, every time I post a new installer it takes a week or two for microsoft to stop false alarm.

**Comment by mgth** [comment]
Stange, I tested it with virus total and Microsoft is ok.
![image](https://github.com/mgth/LittleBigMouse/assets/2568124/cd01b0fe-5954-4d5c-b956-590406ca299c)

**Reply by ChristopherHaws** [reply]
It worked today. Maybe it got added faster than 1 week this time :) Thanks!

**Comment by Constrat** [comment]
I had the same experience. With defender it's a hit or miss. I opened a claim on Microsoft website and they didn't care, another member did, and they accepted it.

p.s. wanted to thank you for the amazing program.

**Comment by SirLemon13** [comment]
I have the program downloaded but it will not open. It shows as running in task manager but the window doesn't open.

**Reply by cmajc** [reply]
I had this same issue and fixed by by going into Windows Security > Firewall & network protection > Allow an app through firewall > then find littlebigmouse.hook and make sure Private and Public are checked. After saving and applying settings when I tried to open Little Big Mouse it worked for me

**Reply by SirLemon13** [reply]
That was a great suggestion, and I was hopeful because it wasn't even listed in the apps. But after adding it and making sure private and public were checked, it still will not open for me. My new monitor is 1440 and old is 1080 and the way they line up is really starting to bug me. I haven't found any other suggestions besides this program to help fix it and I can't even get it to run.

**Reply by Tokou14** [reply]
I have the same issue

**Reply by MavicaCameras** [reply]
I have the same issue in the new released version was a solution presented yet?

**Comment by Aren-R** [comment]
I get this error: "The code execution cannot proceed because MSVCP140.dll was not found. Reinstalling may help". Reinstalled multiple times with no luck

**Comment by Nokikuji** [comment]
This straight up didn't work for me. When I downloaded it and tried to install it, windows defender came up an said it protected my PC. No issue, I clicked more and run anyway. When the installer popped up, I went through the menus and it asked me to download .NET. The download button didn't work so I had to go to Microsoft's website and download/install it myself, after which BigLittleMouse finally installed. When I go to run the application, nothing happens. Nothing shows up, no error, nothing. I ran it as administrator and still nothing happened. Any ideas on what to do to fix this? I'm on Windows 11.

**Reply by Tyler-Petrov** [reply]
I had that problem, but for me LBM opened silently in the system tray.  You may need to right click or double click to get it to work.
![Screenshot 2024-04-17 204144](https://github.com/mgth/LittleBigMouse/assets/52420693/23e8e42c-d507-4a37-be06-d83eed1f4770)
